apps/sms/views/shared/test/unit/activity_handler_test.js

```diff
@@ -353,6 +353,10 @@ suite('ActivityHandler', function() {
         this.sinon.stub(Contacts, 'findByAddress').returns(findByPromise);
 
         MockNavigatormozApps.mTriggerLastRequestSuccess();
+
+        findByPromise.then(() => {
```

Comment:
I thought we might not need this in setup originally since the test below already check after the contact promise resolved, but I was wrong since it didn't "promise" the title promise will start earlier than here. It seems we can not guarantee when the title promise will be finished and this duplicate then after `findByPromise` is just to delay the chain(defiantly a bad idea for testing), or you have other idea about rewrite this test?